SA/design_concept/content/design/non_functional_requirements.typ

```diff
@@ -24,7 +24,7 @@ To find relevant #ac("NFR")s, ISO-25010 @iso-25010 was used for inspiration.
         No extra effort should be required to run VisualFP on any particular #ac("OS").
     ],
     "Test the usage of VisualFP on all three mentioned operating systems.",
-    "The installation steps are the same or of equivalent effort for all three mentioned operating systems",
+    "The installation steps are the same or of equivalent effort for all three mentioned operating systems.",
```

Comment:
Do we want to use punctuation? Because if so, we'll have to do it consistently in all texts (at least, in this table)